### ISSUE TITLE
No outer `mzQC` element

### ISSUE THREAD

**Issue by bittremieux** [MS-Quality-hub/pymzqc#14]
Serializing an `MzQCFile` object doesn't include the outer `mzQC` tag. I.e. the generated result is:

```
{"creationDate": "2021-06-01T17:08:20",
 "version": "1.0.0",
 "runQualities": [],
 "setQualities": [],
 "controlledVocabularies": [{"name": "Proteomics Standards Initiative Quality Control Ontology", "uri": "https://raw.githubusercontent.com/HUPO-PSI/mzQC/master/cv/qc-cv.obo", "version": "0.1.2"}, {"name": "Proteomics Standards Initiative Mass Spectrometry Ontology", "uri": "https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo", "version": "4.1.53"}]
}
```

And should be:

```
{"mzQC": {
    "creationDate": "2021-06-01T17:08:20",
    "version": "1.0.0",
    "runQualities": [],
    "setQualities": [],
    "controlledVocabularies": [{"name": "Proteomics Standards Initiative Quality Control Ontology", "uri": "https://raw.githubusercontent.com/HUPO-PSI/mzQC/master/cv/qc-cv.obo", "version": "0.1.2"}, {"name": "Proteomics Standards Initiative Mass Spectrometry Ontology", "uri": "https://raw.githubusercontent.com/HUPO-PSI/psi-ms-CV/master/psi-ms.obo", "version": "4.1.53"}]}
}
```

**Comment by mwalzer** [comment]
fix in [28e3629](https://github.com/bigbio/mzqc-pylib/pull/12/commits/28e3629431f7e892990c05614901e0d07d21a370)

**Comment by mwalzer** [comment]
fix now in main
